recipes/crms2map/meta.yaml

```diff
@@ -0,0 +1,108 @@
+{% set name = "crms2map" %}
+{% set version = "1.0" %}
+
+package:
+  name: {{ name |lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/jinikeda/{{ name |lower }}/archive/refs/tags/{{ version }}.tar.gz
+  sha256: 7c718dbe50dd3fe7cb2cf294418e846028126ed6a4f9c3960db92b7cfce3ae53
+
+build:
+  # Uncomment the following line if the package is pure Python and the recipe is exactly the same for all platforms.
```

Comment:
Please remove all comments that aren't specific to this recipe

Reply:
Removed all comments that are not related to the recipe.